lighthouse-core/audits/content-width.js

```diff
@@ -55,8 +55,8 @@ class ContentWidth extends Audit {
       return '';
     }
 
-    return 'The content scroll size is ' + artifact.scrollWidth + 'px, ' +
-        'whereas the viewport size is ' + artifact.viewportWidth + 'px.';
+    return 'The content scroll size is ' + artifact.innerWidth + 'px, ' +
+        'whereas the viewport size is ' + artifact.outerWidth + 'px.';
```

Comment:
Ok talked to Paul and found the motivation of this audit and it's just actually measuring something totally different. I'll clean that up in a future PR :) Will rename for now.